versions-common/src/main/java/org/codehaus/mojo/versions/api/DefaultVersionsHelper.java

```diff
@@ -273,11 +274,15 @@ public ArtifactVersions lookupArtifactVersions(
                 repositories = emptyList();
             }
 
+            // in future sessions may become AutoCloseable, so all this would go into try-with-resource block
```

Comment:
This should be marked with TODO then

Reply:
unsure, i just left it here. IMHO is not TODO as session is NOT YET AutoCloseable, may be 1.11.x or maybe 2.0 but nobody can tell WHEN it will be. OTOH, "TODO" would mean "do it if you can", but it cannot be done today.

Reply:
NOTE?